example/basic/example.dart

```diff
@@ -21,10 +21,13 @@ main() {
         name: 'one',
         defaultRoute: true,
         path: '/one',
-        pageTitle: 'Route One',
+        pageTitle: (_) => 'Route One',
```

Comment:
this seemed to be the simplest way to support both simple page titles and complex dynamically generated ones

i'm open to other api suggestions though